src/services/fileStorage/strategies/awsS3.js

```diff
@@ -307,13 +307,13 @@ class AWSS3Strategy extends AbstractFileStorageStrategy {
 				Key: flatFileName,
 				Expires: 60,
 			};
-
-			if (download) {
+			const getBoolean = (value) => value === true || value === 'true';
+			if (getBoolean(download)) {
 				params.ResponseContentDisposition = `attachment; filename = "${localFileName.replace('"', '')}"`;
 			}
-
+			logger.info(params.ResponseContentDisposition);
```

Comment:
Soll die Logausgabe für alle Dateien drin bleiben?

Reply:
Kann raus denke ich.. auch wenn das Kohleveredlung nur in Dev Systemen gesetzt sein sollte

Reply:
is removed, done